fmom/ops_test.go

```diff
@@ -390,3 +394,33 @@ func TestBoostOf(t *testing.T) {
 		})
 	}
 }
+
+func TestVecOf(t *testing.T) {
+	for _, tc := range []struct {
+		p    P4
+		want r3.Vec
+	}{
+		{
+			p:    newPxPyPzE(NewPxPyPzE(0, 10, 10, 20)),
```

Comment:
```suggestion
			p:    newPxPyPzE(NewPxPyPzE(0, 10, 20, 30)),
```

just to address any possible ambiguity.